language/functional_tests/tests/testsuite/payments/print_counterfeit_money.mvir

```diff
@@ -48,6 +48,4 @@ main() {
     return;
 }
 
-// check: TYPE_ERROR
-// check: TYPE_ERROR
-// check: LINKER_ERROR
+// check: MISSING_DATA
```

Comment:
this should be the right error right? I mean I know it is because the test passes but it's also the expected one, is it?

Reply:
@vgao1996 should know for sure, but this looks right to me.